.github/workflows/golangci-lint.yml

```diff
@@ -13,6 +13,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v1
+        uses: golangci/golangci-lint-action@v2
```

Comment:
Nice finding. I will actualize the golangci-lint integration in a separate PR after that. Thanks!

Reply:
(or perhaps before, but yes, I agree this change doesn't belong lumped in here. I just wanted the automated tests passing)